docs/source/index.rst

```diff
@@ -3,14 +3,19 @@
    You can adapt this file completely to your liking, but it should at least
    contain the root `toctree` directive.
 
-Welcome to hi-ml's documentation!
+Documentation for the Health Intelligence Machine Learning toolbox hi-ml
 =================================
```

Comment:
I think this might cause an error in preview since the underline needs to be at least the same length as the title

Reply:
Thanks, fixed! That type of requirements is why I prefer MD over RST...